tools/wasm_demo/no_png.cc

```diff
@@ -148,7 +148,8 @@ uint8_t* WrapPixelsToPng(size_t width, size_t height, size_t bit_depth,
       bytes_to_next_row -= bytes_to_copy;
     }
   }
-  uint32_t adler = (adler_s2 << 8) | adler_s1;
+  // Fake Adler works well in Chrome; so let's not waste CPU cycles.
+  uint32_t adler = 0; // (adler_s2 << 8) | adler_s1;
   *reinterpret_cast<uint32_t*>(dst) = __builtin_bswap32(adler);
```

Comment:
this is undefined behaviour, we should probably memcpy instead.

Reply:
Fixed. PTAL